{{cookiecutter.project_slug}}/compose/production/django/Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR ${APP_HOME}
 
 COPY ./package.json ${APP_HOME}
 RUN npm install && npm cache clean --force
-COPY . ${APP_HOME}
+ADD . ${APP_HOME}
```

Comment:
Could you explain why the COPY command was replaced with ADD in the Dockerfile?
What specific problem does this change aim to solve?

Reply:
The https://github.com/astral-sh/uv-docker-example uses ADD, I don't know the difference.

Reply:
The official documentation on docker ADD vs COPY best practices and some usage examples is very enlightening:

https://docs.docker.com/build/building/best-practices/#add-or-copy

Reply:
I would also suggest to change from:

```
ADD . ${APP_HOME}
```

to: 
```
COPY ./pyproject.toml ./uv.lock ./
```

in the local django Dockerfile

Reply:
I would also suggest to change from:

```
ADD . ${APP_HOME}
```

to: 
```
COPY ./pyproject.toml ./uv.lock ./
```

in the local django Dockerfile

Reply:
Can you elaborate a little more on your requested change?

Reply:
Reverted the change, using COPY in favor of ADD